src/transaction/locator_sr.c

```diff
@@ -13937,6 +13941,166 @@ has_errors_filtered_for_insert (std::vector<int> error_filter_array)
 }
 // *INDENT-ON*
 
+
+static inline void
+locator_initialize_classname_entry (LOCATOR_CLASSNAME_ENTRY * entry, const char *classname, const OID * classoid)
```

Comment:
I think we can merge the "alloc" and this "initialization" since they are always together, and this function doesn't return anything. `locator_alloc_classname_entry`

Reply:
On second thought, it seems better to split it. Sorry, forget this.